### ISSUE TITLE
Set apart the fork from the upstream with updated branding and version

### ISSUE THREAD

**Issue by xmfcx** [autowarefoundation/AWSIM#35]
Now that the fork has started to become very distinct from the original upstream, we need to separate the versioning and the naming to avoid confusing the users.

We can open the new sub brand name to discussion. Here are my proposals for the new name:
- AWSIM Frontier
- AWSIM Spectrum
- AWSIM Horizon
- AWSIM AWF Edition
- AWSIM Labs

~I like **AWSIM Frontier** the most but would like to hear your opinions too.~

Nobody presented any opinions, to keep things simple, We've decided to go with **AWSIM Labs**.

Definition of done:
- [x] Decide on a with a sub-brand name
- [x] Update the documentation with it

**Comment by mozhoku** [comment]
Added AutowareLabs logo with this PR:
- https://github.com/autowarefoundation/AWSIM/pull/39

**Comment by mimi89999** [comment]
Actually, what's the reason why this fork was created? Shouldn't features be proposed upstream first?

**Comment by xmfcx** [comment]
@mimi89999 We had opened several relatively simple PRs in the https://github.com/tier4/AWSIM but they were not reviewed. Also the current maintainers of the upstream AWSIM are way too conservative and it slows down the development progress.

For this reason, we have forked away, sort of a hard fork. We now use URP and they still use HDRP. The performance gap became quite large and we have also added some useful small features too.

We have also switched to the latest Unity LTS and also set up the [dotnet-format](https://github.com/dotnet/format) with pre-commit to automate basic code linting. 

The diffs have grown very large and I don't think we will ever become one with the upstream in near future.
